docs/polkadot/Polkadot-OverviewPaper/background.tex

```diff
@@ -8,45 +8,32 @@ \section{Glossary and Background}
+    BABE & A mechanism to assign elected validators randomly to block production for a certain slot. && \ref{sec:babe} \\
+    BABE Slot & A period for which a relay chain block can be produced. & \slot & \ref{sec:babe} \\
+    Collator & Assist validators in block production. A set of collators is defined as \Col . & \col & \ref{par:collators} \\
+    DOT & The Polkadot native token. && \ref{sec:economics} \\
+    Elected\newline- validators & A set of elected validators. & \Val & \\
+    Epoch & A period for which randomness is generated by BABE. & \ep & \\
+    Era & A period for which a new validator set is decided. && \\
+    Extrinsics & Input data supplied to the Relay Chain to transition states. && \ref{par:extrinsics} \\
+    Fishermen & Monitors the network for misbehavior. && \ref{par:fishermen} \\
+    Gossiping & Broadcast every newly received message to peers. && \ref{sec:gossiping} \\
+    GRANDPA & Mechanism to finalize blocks. && \ref{sec:grandpa} \\
+    GRANDPA\newline- Round & a state of the GRANDPA algorithm which leads to block finalization. && \ref{sec:grandpa} \\
+    Nominator & Stake-holding party who nominates validators to be elected. A set of nominators is defined as \Nom . & \nom & \ref{par:nominators} \\
+    NPoS & \emph{Nominated Proof-of-Stake} - Polkadot's version of PoS, where nominated validators get elected to be able to produce blocks. && \ref{sec:validators} \\
+    Parachain & Heterogeneous independent chain. & \Par & \\
+    PJR & \emph{Proportional-Justified-Representation} - Ensures that validators represent as many nominator minorities as possible. && \ref{par:decentralization} \\
+    PoS & \emph{Proof-of-Stake} - Alternative to PoW, where parties vote with locked funds. && \ref{sec:validators} \\
+    PoV & \emph{Proof-of-Validity} - Mechanism where a validator can verify a block without having its full state. && \ref{sec:parachainblockproduction} \\
+    PoW & \emph{Proof-of-Work} - Mechanism where parties vote with processing power. && \\
+    Relay\newline- Chain & Ensures global consensus among parachains. && \ref{sec:relaychain} \\
+    Runtime & The Wasm blob which contains the state transition functions, including other core operations required by Polkadot. && \ref{par:state_transition} \\
+    Sentry\newline- nodes & Specialized proxy server which forward traffic to/from the validator. && \\
+    STVF & \emph{State-Transition-Validation-Function} - A function of the Runtime to verify the PoV. && \ref{sec:parachainblockproduction} \\
+    Validator & The elected and highest in charge party who has a chance of being selected by BABE to produce a block. A set of candidate validators is defined as \Can . The number of validators to elect is defined as \nval . & \val & \ref{par:validators} \\
+    VRF & \emph{Verifiable-Random-Function} - Cryptographic function for determining elected validators for block production. && \ref{sec:babe} \\
```

Comment:
We use VRFs for many things.  You'll find a better but too long definition in https://github.com/w3f/research/blob/master/docs/papers/sass/sass-1-vrfs.tex